src/TRestRawSignalEvent.cxx

```diff
@@ -354,6 +354,15 @@ Double_t TRestRawSignalEvent::GetMaxTime() {
     return maxTime;
 }
 
+// Specific variable to recover the T2-T1 distance from TRestRawBiPoToSignalProcess
+// to then be used in the new TRestRawBiPoAnalysisProcess
+
+Double_t fAuxiliar = 0;
+
+void SetAuxiliar(Double_t aux) { fAuxiliar = aux; }
+
+auto GetAuxiliar() { return fAuxiliar; }
```

Comment:
Better move to the header file `TRestRawSignalEvent.h`. It should be placed in the `public` region.

Reply:
These methods should be removed

Reply:
I think you still need to remove `GetAuxiliar` here. It is replicated in the header. `fAuxiliar`and `GetAuxiliar`should be removed if they are already in the header.